cmd/context/delete.go

```diff
@@ -28,12 +29,18 @@ import (
 func DeleteCMD() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "delete",
-		Args:  utils.ExactArgsAccepted(1, "https://okteto.com/docs/reference/cli/#delete"),
-		Short: "Delete a context",
+		Args:  utils.MinimumNArgsAccepted(1, "https://okteto.com/docs/reference/cli/#delete"),
```

Comment:
it would be nice to update the docs reflecting this change

Reply:
Hi @andreafalzetti , is the update of docs supposed to be done on my side? If so, are you able to point me in the right direction to do it because I am not too sure. Thanks in advance!

Reply:
up to you really, the docs is open source too: https://github.com/okteto/docs

I've left the comment so at least we don't forget to do it